tests/test_main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_age_should_be_positive_and_round():
```

Comment:
Could you create two separate tests for these cases?

Reply:
Originally this test was a section of two tests. But I thought they could be combined. I'll split it back up.